src/helpers/figures/charts/chart_common_line_scatter.ts

```diff
@@ -224,17 +225,9 @@ export function createLineOrScatterChartRuntime(
   const config = getDefaultChartJsRuntime(chart, labels, fontColor, options);
 
   const legend: DeepPartial<LegendOptions<"line">> = {
+    ...INTERACTIVE_LEGEND_CONFIG,
```

Comment:
Yes, because we don't want it in the case of the scatter plot (we need to draw circle, which are drawn another way), so it's not needed here and added for the line in the line_chart.ts file